test/integration-test.sh

```diff
@@ -449,6 +486,7 @@ if ! [ -z $additional_nodes_addrs ] && ! [ -z $additional_nodes_peerids ]; then
   native_addrs_to_register+=",${additional_nodes_addrs}"
```

Comment:
Please quote `additional_nodes_addrs` and `additional_nodes_peerids` one line up.